Kudu.Core/Deployment/Generator/ExternalCommandBuilder.cs

```diff
@@ -85,5 +114,94 @@ protected void RunCommand(DeploymentContext context, string command)
                 throw;
             }
         }
+
+        /// <summary>
+        /// Mark public for testing
+        /// </summary>
+        public List<string> GetPostBuildActionScripts()
+        {
+            List<string> scriptFilesGroupedAndSorted = new List<string>();
+            List<string> scriptFolders = new List<string>();
+
+            // "/site/deployments/tools/PostDeploymentActions" (can override with %POST_DEPLOYMENT_ACTIONS_DIR%)
+            var defaultPath = Path.Combine(Environment.DeploymentToolsPath, PostDeploymentActions);
+            if (!string.IsNullOrEmpty(System.Environment.GetEnvironmentVariable("POST_DEPLOYMENT_ACTIONS_DIR")))
```

Comment:
Actually, we already have a const for it: `WellKnownEnvironmentVariables.PostDeploymentActionsDirectoryKey`.